tests/Internal/DecimalTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Brick\Validation\Tests\Internal;
+
+use Brick\Validation\Internal\Decimal;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit tests for class Decimal.
+ */
+class DecimalTest extends TestCase
+{
+    /**
+     * @expectedException              \InvalidArgumentException
+     * @expectedExceptionMessageRegExp /Decimal number out of range for this platform: \d+/
+     */
+    public function testParseWithValueIsOutOfRange()
+    {
+        Decimal::parse('1000000000000000000000000');
+    }
+
+    /**
+     * @expectedException        \RangeException
+     * @expectedExceptionMessage The subtraction result overflows.
+     */
+    public function testMinusWithSubstractionResultOverflows()
```

Comment:
Typo: Subtraction

Reply:
Good! I I will fix the correct test method.